gp-cli/src/main/java/com/ibm/g11n/pipeline/tools/cli/CreateBundleCmd.java

```diff
@@ -31,7 +31,7 @@
 @Parameters(commandDescription = "Creates a new translation bundle.")
 final class CreateBundleCmd extends BundleCmd {
     @Parameter(
-            names = {"-l", "--language"},
+            names = {"-l", "--languages"},
```

Comment:
Good point. Wanted to correct long alias and make it plural. But, I realized it's an incompatible change. Although, short one "-l" is likely used in most of cases.